src/style.css

```diff
@@ -0,0 +1,142 @@
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  
+  width: 200px;
+  height: 406px;
```

Comment:
Checklist p.8, don't hardcode height, content should dictate it
<img width="1054" alt="image" src="https://user-images.githubusercontent.com/118751034/213753197-d734c0c3-084b-4b78-8511-6ae7deb44459.png">